charts/argo-workflows/templates/controller/artifact-repository-ref-cm.yaml

```diff
@@ -0,0 +1,19 @@
+{{- range $cm_name, $cm_val := .Values.artifactRepositoryRef }}
```

Comment:
Not list but map is intentional to merge element easily.
If reviewers want, I can change this to list.